snap/quota/quota.go

```diff
@@ -303,6 +304,18 @@ func (grp *Group) JournalServiceDropInFile() string {
 	return filepath.Join(grp.JournalServiceDropInDir(), "00-snap.conf")
 }
 
+// FindQuotaGroupForServiceInSubGroups returns the quota group for the given
+// service name if found in one of the sub-groups of the current group. If the
+// service is not found, this will return nil.
+func (grp *Group) FindQuotaGroupForServiceInSubGroups(serviceName string) *Group {
```

Comment:
this maybe should just be called QuotaGroupForService or GroupForService?

Reply:
Ah nice suggestion!